### PR TITLE
Get all notifications

### PR DESCRIPTION
**Pull request by Nixolay**
Pull request created for fix moira-alert/moira 236.

### CODE REVIEW

File: src/Components/SubscriptionEditor/SubscriptionEditor.jsx [hunk 1]
```diff
@@ -59,7 +60,7 @@ export default class SubscriptionEditor extends React.Component<Props> {
                 </div>
                 <div className={cn("row")}>
                     <div className={cn("caption")}>
-                        Tags:{" "}
+                        Tags asdasdasd:{" "}
```
Comment (sashasushko) [inline src/Components/SubscriptionEditor/SubscriptionEditor.jsx]: What is this?

File: src/Components/SubscriptionEditor/SubscriptionEditor.jsx [hunk 2]
```diff
@@ -79,14 +80,27 @@ export default class SubscriptionEditor extends React.Component<Props> {
                                 value={subscription.tags}
                                 onChange={nextTags => {
                                     onChange({
-                                        tags: nextTags,
+                                        tags: subscription.any_tags ? null : nextTags,
```
Comment (sashasushko) [inline src/Components/SubscriptionEditor/SubscriptionEditor.jsx]: Why you need this, if you disabled field?

File: src/Components/SubscriptionEditor/SubscriptionEditor.jsx [hunk 3]
```diff
@@ -79,14 +80,27 @@ export default class SubscriptionEditor extends React.Component<Props> {
...
                             />
                         </ValidationWrapperV1>
                     </div>
                 </div>
+                <div className={cn("row")}>
+                    <span className={cn("any_tags")}>
+                        <Toggle
+                            onChange={checked =>
+                                onChange({
+                                    any_tags: checked,
```
Comment (sashasushko) [inline src/Components/SubscriptionEditor/SubscriptionEditor.jsx]: If you put new prop and then change it, you must describe it in parrent state types. I don't see it

File: src/Components/SubscriptionEditor/SubscriptionEditor.jsx (hunk 1 above)
Comment (Nixolay) [reply src/Components/SubscriptionEditor/SubscriptionEditor.jsx]: Corrected

File: src/Components/SubscriptionEditor/SubscriptionEditor.jsx (hunk 2 above)
Comment (Nixolay) [reply src/Components/SubscriptionEditor/SubscriptionEditor.jsx]: Corrected

File: src/Components/SubscriptionEditor/SubscriptionEditor.jsx (hunk 3 above)
Comment (Nixolay) [reply src/Components/SubscriptionEditor/SubscriptionEditor.jsx]: How to do it?

**Comment by sashasushko** [comment]
@Nixolay а это что за восстановленные коммиты от `Nix O Live`?

**Comment by sashasushko** [comment]
@Nixolay почему закрыл?

**Pull request closed without merging**